.changeset/hip-eels-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit': patch
+'lit-html': patch
+---
+
+export StaticValue type
```

Comment:
Nit: Could this be turned into a full sentence as it'll appear in the changelog.md